config/manifests/bases/percona-server-mysql-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+  name: percona-server-mysql-operator.v0.0.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: PerconaServerMySQLBackup is the Schema for the perconaservermysqlbackups
+        API
+      displayName: Percona Server My SQLBackup
```

Comment:
`Percona Server My SQLBackup` -> `Percona Server MySQL Backup` or something like that.

Reply:
oh, that shouldn't be here. I'll remove it